### PR TITLE
WIP: Added endpoint to get statistics about the Dispersy communities

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

**Comment by devos50** [comment]
@lfdversluis can you review?

File: Tribler/Core/Modules/restapi/debug_endpoint.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+class DebugCommunitiesEndpoint(resource.Resource):
+    """
+    A GET request to this endpoint will return statistics about the loaded communities in Tribler.
+
+    Example response (partially since the full response it too large to display):
+    {
+        "communities": {
+            "<ChannelCommunity>: b9754da88799ff2dc4042325bd8640d3a5685100": {
+                "Sync bloom created": "8",
```
Comment (lfdversluis) [inline Tribler/Core/Modules/restapi/debug_endpoint.py]: Do we want spaces in a key? Isn't underscores more usual?

File: Tribler/Core/Modules/restapi/debug_endpoint.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                "Statistics": {
+                    "outgoing": {
+                        "-caused by missing-proof-": "1",
```
Comment (lfdversluis) [inline Tribler/Core/Modules/restapi/debug_endpoint.py]: why does this start with a dash?

File: Tribler/Core/Modules/restapi/debug_endpoint.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                        "dispersy-puncture-request": "8",
```
Comment (lfdversluis) [inline Tribler/Core/Modules/restapi/debug_endpoint.py]: this uses dashes instead of spaces, not consistent

**Comment by devos50** [comment]
Will open a new PR with these changes later.

**Pull request closed without merging**